Artifacts/windows-appxremove/remove-appx-packages.ps1

```diff
@@ -0,0 +1,60 @@
+﻿###################################################################################################
+#
+# PowerShell configurations
+#
+
+# NOTE: Because the $ErrorActionPreference is "Stop", this script will stop on first failure.
+#       This is necessary to ensure we capture errors inside the try-catch-finally block.
+$ErrorActionPreference = "Stop"
+
+# Hide any progress bars, due to downloads and installs of remote components.
+$ProgressPreference = "SilentlyContinue"
+
+# Discard any collected errors from a previous execution.
+$Error.Clear()
+
+# Allow certian operations, like downloading files, to execute.
+Set-ExecutionPolicy Bypass -Scope Process -Force
+
+###################################################################################################
+#
+# Handle all errors in this script.
+#
+
+trap
+{
+    # NOTE: This trap will handle all errors. There should be no need to use a catch below in this
+    #       script, unless you want to ignore a specific error.
+    $message = $Error[0].Exception.Message
+    if ($message)
+    {
+        Write-Host -Object "`nERROR: $message" -ForegroundColor Red
+    }
+
+    Write-Host "`nThe artifact failed to apply.`n"
+
+    # IMPORTANT NOTE: Throwing a terminating error (using $ErrorActionPreference = "Stop") still
+    # returns exit code zero from the PowerShell script when using -File. The workaround is to
+    # NOT use -File when calling this script and leverage the try-catch-finally block and return
+    # a non-zero exit code from the catch block.
+    exit -1
+}
+
+###################################################################################################
+#
+# Main execution block.
+#
+
+try
```

Comment:
`try` is not required, unless you have a `catch` or `finally` block. The `trap` will handle the error for you.

Reply:
done